### DISCUSSION TITLE
TSML Colors

### DISCUSSION THREAD

**Discussion by puebloaa** [code4recovery/12-step-meeting-list#1536] [General]
When I originally installed TSML,  I got some CSS code to match the colors with those of our site.  I looked at the page recently and realized that the text color on the gray stripe is unreadable.  I was wondering if there is some code I can add to make the lettering white so it is more legible.

The current CSS is:
#tsml .table-striped>tbody>tr:nth-of-type(odd) { background-color: #415e75; }
#tsml a { color: white }

The URL for the page is:
https://www.puebloaa.org/meetings/

Thank you in advance for your assistance and for all the work you do for the recovery community.

Kevin

**Comment by gkovats** [comment]
Maybe try 
```css
#meetings_tbody { background: #fff; }
```
and remove the current - the blue background is the challenge.
![image](https://github.com/user-attachments/assets/62308f72-81fd-4581-ac4f-bda6a58e7e54)

**Comment by joshreisner** [comment]
☝️ that works, or if you just want to set the color of the gray rows white, change your existing code to 
```
#tsml .table-striped>tbody>tr:nth-of-type(odd) { background-color: #415e75; color: white; }
#tsml a { color: white }
```

**Comment by puebloaa** [comment]
That got it, thanks for your help.

Kevin
